buildSrc/src/main/kotlin/otel.publish-conventions.gradle.kts

```diff
@@ -0,0 +1,63 @@
+plugins {
+  `maven-publish`
+  signing
+}
+
+publishing {
+  publications {
+    register<MavenPublication>("mavenPublication") {
+      groupId = "io.opentelemetry.proto"
```

Comment:
Important

Reply:
@jkwatson The coordinates don't collide because this group is different.

Reply:
cool. yes, you are correct. thanks for re-iterating!